docs/eventing/broker/README.md

```diff
@@ -1,111 +1,33 @@
-A Broker represents an 'event mesh'. Events are sent to the Broker's ingress and
-are then sent to any subscribers that are interested in that event. Once inside
-a Broker, all metadata other than the CloudEvent is stripped away (e.g. unless
-set as a CloudEvent attribute, there is no concept of how this event entered the
-Broker).
-
-There can be different classes of Brokers providing different kinds
-of semantics around durability of events, performance, etc. The Broker that is
-part of the Knative Eventing repo is used for these examples, it uses Knative
-[Channels](../channels/) for delivering events. You can read more details about
-[MT Channel Based Broker](./mt-channel-based-broker.md). Simple example showing a `Broker`
-where the configuration is specified in a `ConfigMap` config-br-default-channel,
-which uses `InMemoryChannel`:
-
-Example:
+Knative provides a multi-tenant, channel-based broker implementation that uses channels for event routing.
 
-```yaml
-apiVersion: eventing.knative.dev/v1
-kind: Broker
-metadata:
-  annotations:
-    eventing.knative.dev/broker.class: MTChannelBasedBroker
-  name: default
-spec:
-  # Configuration specific to this broker.
-  config:
-    apiVersion: v1
-    kind: ConfigMap
-    name: config-br-default-channel
-    namespace: knative-eventing
-```
+Before you can use the Knative Broker, you must install a channel provider, such as InMemoryChannel, Kafka or Nats.
```

Comment:
Do we want to decide on a consistent way to refer to the Broker maybe across docs? I wonder about whether we mention it's channel-based, multitenant, etc, because it's referred to in different ways at the minute, maybe it's just Knative Channel-based Broker though :slightly_smiling_face: again, good feedback!
cc @lionelvillard @matzew

Reply:
I like Knative Channel-based Broker.

I think the fact that it's a shared data plane (what we're calling multi-tenant) is not so important to the user. Channel-based is important so I would keep that qualifier. As other broker implementations are developed and documented, we'll need to be clear that it's the Channel-based broker being referred to here.

One important note is that the Channel-based Broker is the default Broker implementation for Knative, but not all Broker implementations are Channel-based (and some distributions of Knative may have a different Broker implementation as default). In conceptual documentation, I think we should refer to Broker in the abstract without referencing a specific implementation.